sunpy/map/sources/psp.py

```diff
@@ -20,17 +23,43 @@ class WISPRMap(GenericMap):
     * `WISPR Instrument Page <https://wispr.nrl.navy.mil//>`__
     * `Instrument Paper <https://doi.org/10.1007/s11214-014-0114-y>`__
     """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+
+        self.plot_settings['norm'] = ImageNormalize(
+            stretch=AsinhStretch(a=0.001), clip=True,
+            vmin=0)
+
     @property
     def processing_level(self):
         lvl = self.meta.get('level', None)
         if lvl is None:
             return
-        return int(lvl[1])
+        # Chop off the leading 'L'
+        lvl = lvl[1:]
```

Comment:
```suggestion
        # Chop off the leading 'L' if present
        lvl = lvl[1:] if lvl[0] == 'L' else lvl
```
Since you are allowing for custom levels, it seems odd not to check for whether the character being popped off is 'L' or not.